File: skills/csharp/experimental/musicskill/Models/SkillState.cs
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

using Luis;

namespace MusicSkill.Models
{
    public class SkillState
    {
        public string Token { get; set; }

        public MusicSkillLuis LuisResult { get; set; }

        public void Clear()
        {
        }
    }
}

File: templates/Virtual-Assistant-Template/typescript/samples/sample-skill/src/models/skillState.ts

/**
 * Copyright(c) Microsoft Corporation.All rights reserved.
 * Licensed under the MIT License.
 */
import { RecognizerResult } from 'botbuilder';

export class SkillState {
    public readonly token: string = '';
    public luisResult: RecognizerResult | undefined;
    // eslint-disable-next-line @typescript-eslint/no-empty-function
    public clear(): void {}
}
